cmd/priceFeeder/main.go

```diff
@@ -0,0 +1,200 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-eth-bridge/config"
+	elrondCore "github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-crypto/signing"
+	"github.com/ElrondNetwork/elrond-go-crypto/signing/ed25519"
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/aggregator"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/aggregator/fetchers"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/aggregator/notifees"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/blockchain"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/builders"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/core/polling"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/data"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/interactors"
+)
+
+const (
+	configPath                = "config/config.toml"
+	base                      = "ETH"
```

Comment:
we should get these const values from the configs:
we should have a "general" config part for the poll interval, auto send interval, contract address, network path and such and a price fetcher part that will be an array of structs holding the L27-L31 values